modin/pandas/base.py

```diff
@@ -751,7 +751,9 @@ def count(self, axis=0, level=None, numeric_only=False):
                 # error thrown by pandas
                 raise TypeError("Can only count levels on hierarchical columns.")
 
-            return self._handle_level_agg(axis, level, "count")
+            result = self._handle_level_agg(axis, level, "count")
+            result.sort_index(inplace=True)
+            return result
```

Comment:
`_handle_level_agg` has an internal `sort` parameter which it passes to groupby, maybe we should make that parameter external for `_handle_level_agg` and will pass `sort=True` for `count` besides sorting the result?

Reply:
Sounds reasonable, implemented in the new commit.